src/lib/icons/play_48px.svg

```diff
@@ -1,4 +1,5 @@
-<svg fill="#fff" height="48" viewBox="0 0 24 24" width="48" focusable="false">
+<svg fill="#fff" height="48" viewBox="0 0 24 24" width="48" focusable="false" role="img" aria-labelledby="play-button-icon-01">
```

Comment:
Icons should be unaware of how they're used. It looks like we need to add a hidden span inside the button rather than in the SVG itself.